internal/painter/gl/painter.go

```diff
@@ -11,10 +11,12 @@ import (
 )
 
 var shaderSources = map[string][2][]byte{
-	"line":      {shaderLineVert.StaticContent, shaderLineFrag.StaticContent},
-	"line_es":   {shaderLineesVert.StaticContent, shaderLineesFrag.StaticContent},
-	"simple":    {shaderSimpleVert.StaticContent, shaderSimpleFrag.StaticContent},
-	"simple_es": {shaderSimpleesVert.StaticContent, shaderSimpleesFrag.StaticContent},
+	"line":         {shaderLineVert.StaticContent, shaderLineFrag.StaticContent},
+	"line_es":      {shaderLineesVert.StaticContent, shaderLineesFrag.StaticContent},
+	"simple":       {shaderSimpleVert.StaticContent, shaderSimpleFrag.StaticContent},
+	"simple_es":    {shaderSimpleesVert.StaticContent, shaderSimpleesFrag.StaticContent},
+	"rectangle":    {shaderRectangleVert.StaticContent, shaderRectangleFrag.StaticContent},
+	"rectangle_es": {shaderRectangleesVert.StaticContent, shaderRectangleesFrag.StaticContent},
 }
```

Comment:
You might also want to change this to a switch statement in line with https://github.com/fyne-io/fyne/pull/3244. I won't force you but it seems better than to have me wait for your PR, or not wait and create merge conflicts here :)

Reply:
Please have a look at the coding.